pkg/config/config.go

```diff
@@ -52,6 +53,7 @@ func new() *Config {
 		HttpPort:       getEnvAsInt("HTTP_PORT", 4010),
 		PlaygroundMode: getEnvAsBool("PLAYGROUND_MODE", false),
 		QueryLimit:     getEnvAsInt("QUERY_LIMIT", 1000),
+		SlowLog:        getEnvAsInt("SLOW_LOG", int(300)),
```

Comment:
Why are we setting default to 300 ms? Will we adjust this once we have better idea of funcs average run time?

Reply:
Yes, 300ms is my initial guess, we'll adjust as we learn more. 
I also added an env variable `SLOW_LOG` in case we need to adjust for a specific cluster.